docs/index.yaml

```diff
@@ -5,6 +5,19 @@ page:
   siteTitle: Cardano Wallet
   headHtml: |
     <snippet var="js.fixAnchorLinks" />
+    <style>
+      p code, li code {
+        background-color: rgb(243,244,246) !important;
+        color: black !important;
+      }
+      pre {
+        background-color: rgb(243,244,246) !important;
+      }
+    </style>
+  # NOTE: The template styles <code> tags by adding classes
+  # to the `class` attribute.
+  # I did not find a way to configure this, hence the
+  # CSS styling hack above.
```

Comment:
Yeah. :confused:  This could improve in future versions of emanote... I hope. The other thing is that if `prism.js` is enabled on a page, then it will take over styling of a `<pre><code>` block.